plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -11,6 +11,7 @@ import ServiceResumeModal from "./ServiceResumeModal";
 import ServiceScaleFormModal from "./ServiceScaleFormModal";
 import ServiceSpecUtil from "../../utils/ServiceSpecUtil";
 import ServiceSuspendModal from "./ServiceSuspendModal";
+import GroupContainsFrameworksModal from "./GroupContainsFrameworksModal";
```

Comment:
Please consider how we could rename this to prevent binding to the reason it's invoked. Perhaps `DisabledGroupDestroyModal`.

Reply:
Updated the name